src/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Index Page</title>
+</head>
+<body>
+    <div class="container">
+        <h1>Heading</h1>
+        <section class="grid">
+            <div class="grid-item">
+                <h2>Left Part</h2>
+                <button id="btn">Button</button>
+                <ul id="list"></ul>
+            </div>
+            <div class="grid-item right">
+                <h2>Right Part</h2>
+                <form id="form">
+                    <label for="input1">Input 1:</label>
+                    <input type="text" id="input1" name="input1" required>
+                    <label for="input2">Input 2:</label>
+                    <input type="password" id="input2" name="input2" required>
+                    <input type="submit" value="Submit" class="submit-button">
+                
+                </form>
+            </div>
+        </section>
+    </div>
```

Comment:
- [ ] Kindly match your app structure with the original wireframe. Currently, you are missing some placeholders in your design that do not look like the wireframe. Kindly see the screenshot below. 👍🏽
- [ ] Kindly have all placeholders in the wireframe present in your app structure. 👍🏽

**Your Structure**

<img width="878" alt="Screenshot 2023-05-24 at 5 26 59 PM" src="https://github.com/kayjeee/leaderboard-v2/assets/50941074/48ec92bf-dcbe-483c-b3ba-68ebe8d4185d">


**The Wireframe Structure**

<img width="553" alt="Screenshot 2023-05-24 at 5 25 44 PM" src="https://github.com/kayjeee/leaderboard-v2/assets/50941074/76c5b02e-8d46-4a1e-8662-1c4c45e1a529">

Reply:
made required changes ,please look again